### PR TITLE
Update vendor

### PR DESCRIPTION
**Pull request by dorileo**
This PR updates clui's vendored code and update TUI's code to reflect the clui's theme framework changes.

A couple of linter fixes is added on top of it.

### CODE REVIEW

File: args/args_test.go [hunk 1]
```diff
@@ -42,7 +42,10 @@ func serveHTTPDescFile(t *testing.T) (*http.Server, error) {
 	srv := &http.Server{Addr: ":" + testHTTPPort}
 
 	http.HandleFunc("/clr-installer.yaml", func(w http.ResponseWriter, r *http.Request) {
-		fmt.Fprintf(w, "{}")
+		_, err := fmt.Fprintf(w, "{}")
+		if err != nil {
+			fmt.Printf("%s", err)
```
Comment (mdhorn) [inline args/args_test.go]: Can we use t.Error(err) in the handler?

File: themes/clr-installer.theme [hunk 2]
```diff
@@ -65,23 +65,51 @@ SparkChart=█
...
+MenuButtonBack = blue
+
+MenuButtonActiveText=blue bold
+MenuButtonActiveBack=white
+
+SubMenuButtonText=white
+SubMenuButtonBack=blue
+
+SubMenuButtonActiveText=blue
+SubMenuButtonActiveBack=white
```
Comment (mdhorn) [inline themes/clr-installer.theme]: Can you add a newline to this last line please.

File: args/args_test.go (hunk 1 above)
Comment (dorileo) [reply args/args_test.go]: sure

**Comment by dorileo** [comment]
@mdhorn done. thanks.

**Pull request merged**